StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewController.swift

```diff
@@ -34,6 +34,9 @@ final class UpdatePaymentMethodViewController: UIViewController {
         }
     }
 
+    private var updateCardBrand: Bool = false
+    private var updateDefaultPaymentMethod: Bool = false
```

Comment:
I think these should be in the view model, and maybe named like "hasChangedCardBrand", "hasChangedDefaultPaymentMethodCheckbox.." or something better

Then you could have a computed variable

```swift
    var hasUpdates: Bool {
        return hasChangedCardBrand || hasChangedDefaultPaymentMethodCheckbox
    }
```